src/DGtal/dec/DiscreteExteriorCalculus.h

```diff
@@ -294,13 +316,42 @@ namespace DGtal
     eraseCell(const Cell& cell);
 
     /**
+     * Update indexes for all cells.
+     * Cell insertion order == index may not be preserved.
+     */
+    void
+    updateIndexes();
+
+    /**
      * Get all cells properties.
      * @return associative container from Cell to Property.
      */
-    Properties
+    const Properties&
     getProperties() const;
 
     /**
+     * Get all signed cells with specific dimension and duality in index order.
+     * @tparam order order of signed cells.
+     * @tparam duality duality of signed cells.
+     * @return index ordered signed cells.
+     */
+    template <Order order, Duality duality>
+    const SCells&
+    getIndexedSCells() const;
+
+    /**
+     * Reorder operator from order-forms to order-forms.
```

Comment:
see other memebers description (ie identity or derivative).
order refers to the template parameter.